### PR TITLE
Speed up building and testing in Travis CI

### PR DESCRIPTION
**Pull request by roberthusak**
- Add namespaces to all the tests in `tests` folder so that they are isolated.
- Fix the tests accordingly (mostly by prepending library types by `\` and using the particular namespace in reflection stuff).
- Fix the usage of `__FUNCTION__` inside a namespace.
- Build the tests as one project in Travis CI and only run them one by one - about 4x times faster than compiling them separately.
- Run Peachpie.Runtime.Tests in Travis CI.

### CODE REVIEW

File: tests/arrays/array_map.php [hunk 1]
```diff
@@ -18,8 +19,8 @@ function g($x)
   $d = array('A' => 1,10 => 2);
   
   print_r(array_map(null,$a,$b,$c));
-  print_r(array_map("f",$a,$b,$c));
-  print_r(array_map("g",$d));
+  print_r(array_map("arrays\\array_map\\f",$a,$b,$c));
```
Comment (jakubmisek) [inline tests/arrays/array_map.php]: a pseudoconstant would be nicer `__NAMESPACE__ . "\\f"`

File: tests/program.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+include $_SERVER['argv'][1];
```
Comment (jakubmisek) [inline tests/program.php]: isn't it just `$argv[1]` ?

File: tests/program.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?php
```
Comment (jakubmisek) [inline tests/program.php]: if you'd rename it to `program.phpx` or something, you can get rid of most of the workarounds ;)

File: tests/program.php (hunk 2 above)
Comment (roberthusak) [reply tests/program.php]: It seemed not to work before, but now it does. Maybe it was due to some other reason that I missed.. Anyway, I changed it.

**Comment by jakubmisek** [comment]
thank you @roberthusak !

File: tests/program.php (hunk 3 above)
Comment (roberthusak) [reply tests/program.php]: Seems a bit hacky, but useful indeed :grin:

**Comment by roberthusak** [comment]
@jakubmisek ok, thanks for the review!

**Pull request merged**